client/gutenberg/extensions/presets/jetpack/editor-shared/get-gutenberg-site-fragment.js

```diff
@@ -0,0 +1,32 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import { includes } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import { getSiteFragment } from 'lib/route/path';
+
+/***
+ * Returns the site fragment in the environment we're running Gutenberg in.
+ *
+ * @returns {?Number|String} Site fragment (ID or slug); null for WP.org wp-admin.
+ */
+export default function getGutenbergSiteFragment() {
+	// WP.com wp-admin exposes the site ID in window._currentSiteId
+	if ( window._currentSiteId ) {
```

Comment:
Should we protect against not having `window` available, especially since this is a shared method?

Reply:
👍 Done in https://github.com/Automattic/wp-calypso/pull/28998/commits/06e355600bb4502f0e07ecbcad52dbf299a0b05f